README.md

```diff
@@ -86,6 +86,7 @@ Start a benchmark:
     ./run_benchmark \
         --dsn postgresql://postgres@localhost/<target-db> \
         [--benchmark] <tpch|tpcds|ssb|htap> \
+	--use-named-cursors
```

Comment:
Why is this parameter called use named cursors when the explanation of what it does is use server-side cursors?

Reply:
But this is a user facing parameter, IMHO it should be logically descriptive.